docs/useListController.md

```diff
@@ -304,3 +306,30 @@ const OfficeList = () => {
     );
 };
 ```
+
+## Controlling Access To The List
```

Comment:
```suggestion
## Access Control
```

Let's keep the same title so that readers know where to find that information across hooks and components.